### PR TITLE
Add a !helper command for pinging helpers

### PR DESCRIPTION
**Pull request by Retsam**
Rather than allowing direct pinging of the Helper role, which tends to be 'abused' by people who immediately ping after asking a question, this adds a command that checks if enough time has elapsed (based on a configured environment variable) and pings if so.

A helper (other than the asker) can invoke the command and ignore the timeout, if there's some reason why they think that's appropriate (maybe the question previously closed due to inactivity).

### CODE REVIEW

**Comment by tjjfvi** [comment]
(the changes here are copied more-or-less verbatim to #165)

**Comment by robertt** [comment]
Sorry for not getting to this sooner - two questions:

1. Would it be a good idea to add something about this somewhere in the text after someone claims a channel? I don't think this is intuitive unless mentioned somewhere.
2. How long do we think the timeout should last? 1-2 hours seems reasonable to me

**Review by robertt — APPROVED** [review]

File: src/modules/helpchan.ts [hunk 1]
```diff
@@ -277,6 +277,48 @@ export class HelpChanModule extends Module {
 		await msg.delete({ reason: 'Pin system message' });
 	}
 
+	@command({
+		description: 'Pings a helper in a help-channel',
+		aliases: ['helpers'],
```
Comment (ckiee) [inline src/modules/helpchan.ts:282]:
```suggestion
		aliases: ['helpers'],
		inhibitors: [CommonInhibitors.guildsOnly]
```

Comment (tjjfvi) [reply src/modules/helpchan.ts:282]: Wouldn't that be covered by the `msg.channel.parentID !== categories.ongoing` check?

Comment (ckiee) [reply src/modules/helpchan.ts:282]: It *is* a nitpick (:

**Comment by robertt** [comment]
I'm going to merge this now with a two hour cooldown - we can see how it works and adjust from there

**Pull request merged**